.travis.yml

```diff
@@ -39,7 +39,13 @@ script:
   - |
     if [[ ! -z "$WP_VERSION" ]] ; then
       phpunit
+    fi
+  - |
+    if [[ ! -z "$WP_VERSION" ]] ; then
```

Comment:
The travis config that we use internally is a little different. This is really aimed at future improvements to FM's travis config.